scripts/common/rook.sh

```diff
@@ -205,6 +205,11 @@ function should_upgrade_rook_10_to_14() {
         return 1
     fi
 
+    # this upgrade process does not yet support airgap
+    if [ "$AIRGAP" = "1" ]; then
+        return 1
```

Comment:
can you reword the warning a bit?

```
        echo "Not upgrading rook because it is not installed or the currently installed version is not 1.0.x"
```

Reply:
no because this won't actually come up in tasks.sh yet - it doesn't parse the airgap flag and so will attempt to reach out to the internet